data-models/untp-dte/artefacts/DigitalTraceabilityEvent-schema.json

```diff
@@ -760,17 +760,13 @@
           },
           "description": "An array (one for each sensor) of sensor device data sets associated with the event. "
         },
-        "sourcePartyId": {
-          "example": "https://abr.business.gov.au/ABN/View?abn=90664869327",
```

Comment:
I'm finding out why the example is removed in a Jargon issue (I think it's because we're using a `nestedExample`, as we can no longer use a straight `example`).

Reply:
https://github.com/jargon-sh/issues/issues/56

Reply:
OK, Alastair says he'll update Jargon so we won't lose the example in the JSONSchema. But after re-thinking, I don't think we need to wait for that, necessarily (and it'll be fine as another diff once it is done), so updating to ready for review.